tests/vmi_configuration_test.go

```diff
@@ -1857,8 +1858,10 @@ var _ = Describe("[sig-compute]Configurations", func() {
 	Context("[rfe_id:140][crit:medium][vendor:cnv-qe@redhat.com][level:component]with CPU request settings", func() {
 
 		It("[test_id:3127]should set CPU request from VMI spec", func() {
-			vmi := tests.NewRandomVMI()
-			vmi.Spec.Domain.Resources.Requests[kubev1.ResourceCPU] = resource.MustParse("500m")
+			vmi := libvmi.New(
+				libvmi.WithResourceMemory("1Mi"),
```

Comment:
This is not enough memory for basically VMI, is it on purpose so that it will fail to start?

Reply:
Hey Jed,
Thanks for reviewing
If i understood correctly NewRandomVMI generate diskless vmi so it doesn't need memory.
The reason for the 1Mi is because virt-api doesn't let creating vmi without memory.
The purpose of this is to be more resources efficient.

Reply:
Diskless VMs still need memory, for things like the firmware. I guess it's possible that the emulated BIOS manages to run on such a small amount of memory (EFI definitely wouldn't), but it still feels extremely tight. Can we do something like 32Mi? It remains quite insignificant for any CI infrastructure, and it would ensure future updates don't break tests.

Reply:
Just note: EFI needs a hundred or so if I remember correctly.

Reply:
Please bear in mind that in `template.go`'s `renderLaunchManifest()`, where we convert a VMI manifest into a target pod, we take care (and should take care) of the required memory allocations in order for the desired spec to be possible. It is not the user's responsibility. Any memory that the user requests is accumulated into the memory we allocate automatically.

I'm not sure about EFI specifically, but if we don't automatically add memory for it in `renderLaunchManifest()`, then we should :)

Reply:
@itamarh I believe what you are referring to is the memory allocation for the compute container.
What I'm talking about here is the actual amount of RAM the VM sees, which I don't think we mess with, or do we?

Reply:
@jean-edouard and I were talking offline. Here's a summary of our conclusions.

Currently it's the user's responsibility to allocate the memory needed for the guest, so currently 1MB wouldn't be enough. But this behavior is wrong. The user (or the tests for that matter) shouldn't worry about such low-level-implementation-detail considerations - we should.

We think the solution would be to enforce a minimum RAM amount, and set it as a default.

Until then we can start with `32Mi`, which @jean-edouard suggested as a good starting point.

Reply:
Done.
Thanks!